### ISSUE TITLE
Bug: Dasboard widget missing margins

### ISSUE THREAD

**Issue by enejb** [Automattic/jetpack#2916]
When running the latest version WP (4.4 beta ) I notices that the dashboard widget looks like this 
![screen shot 2015-10-28 at 4 25 05 pm](https://cloud.githubusercontent.com/assets/115071/10806193/b8417f64-7d90-11e5-9418-ae44e2e6ce72.png)

cc @jeffgolenski

**Comment by kraftbj** [comment]
Related:
* #2822 
* https://make.wordpress.org/core/2015/10/28/headings-hierarchy-changes-in-the-admin-screens/
* https://core.trac.wordpress.org/ticket/33558

tl;dr - this is very actively in flux and Core a11y team is aware of the issue with Jetpack. They're looking to add BC CSS for plugins in a situation like ours. They actually had a commit that made it right by us, but it broke other things.

My suggestion is to hold off on this until the dust settles, aim for 3.8.1 pre-4-4 to implement the recs of the make/core post with regard to headings in the widget (before looking closely, we should be able to fix by changing that heading tag to the right one, semantically).
